### DISCUSSION TITLE
Проблемы с ру сайтами, приложениями

### DISCUSSION THREAD

**Discussion by wannaliveinpeace** [bol-van/zapret#838] [Q&A]
День добрый.
Накатил запрет на кинетик гига 1011, ютуб заработал все вроде бы отлично и через время заметил, перестали работать рутюб, авито и дискорд умер. Может есть у кого мысли как починить?
Возможно ли сделать сразу несколько стратегий, которые будут затрагивать сайты рф( с окончанием .ru), дискорд и т.д.?

**Comment by electrifying** [comment]
смотря какая стратегия и режим фильтрации

**Comment by janeblower** [comment]
> hostlist-exclude=<filename>			; не применять дурение к доменам из листа. может быть множество листов.

**Comment by wannaliveinpeace** [comment]
> > hostlist-exclude=			; не применять дурение к доменам из листа. может быть множество листов.

А где это сделать и как? Я немного чайник в данном вопросе, поэтому если чуть подробнее расписали бы, было бы классно

**Comment by electrifying** [comment]
> А где это сделать и как?

Zapret/ipset там файл хостлиста с исключениями
